atomics/T1070.002/T1070.002.yaml

```diff
@@ -24,7 +42,6 @@ atomic_tests:
     command: |
       sudo log erase --all
       sudo log erase --ttl #Deletes only time-to-live log content
-      sudo log erase --predicate 'subsystem == "com.apple.appstore"' #Deletes all logs related to the App Store, useful for clearing specific entries of the system log
```

Comment:
on my Big Sur / Monterey systems, there is no `--predicate` argument to `log erase`...